app/views/accounts/organizations/_teams.html.erb

```diff
@@ -100,6 +100,25 @@
         <% row.with_cell do %>
           <%= member.successful_invite_for(current_organization)&.sender&.email || "–" %>
         <% end %>
+        <% user_role = current_user.role_for(current_organization) %>
+        <% member_role = member.role_for(current_organization) %>
+        <% if ((user_role == "owner" && member_role.in?(["developer", "viewer"])) or
+            (user_role == "developer" && member_role == "viewer")) and current_user.id != member.id %>
```

Comment:
**suggestion**: we generally have components/presenters for complicated view logic, and we've been trying to minimize our usage of view helpers (because they are a bad abstraction) – but it seems to me since this page or controller is relatively simple, so we can move this bit of rather involved logic to a view helper, perhaps an `OrganizationsHelper` with `can_edit_user_role`?